include/h2o/http3_common.h

```diff
@@ -119,10 +119,21 @@ size_t h2o_http3_goaway_frame_capacity(quicly_stream_id_t stream_or_push_id);
 uint8_t *h2o_http3_encode_goaway_frame(uint8_t *buff, quicly_stream_id_t stream_or_push_id);
 int h2o_http3_decode_goaway_frame(h2o_http3_goaway_frame_t *frame, const uint8_t *payload, size_t len, const char **err_desc);
 
+/*
+ * Special dummy values for representing errors from http3 acceptor
+ */
+extern h2o_quic_conn_t h2o_quic_accept_conn_decryption_failed;
+extern h2o_http3_conn_t h2o_quic_accept_conn_closed;
+
 /**
  * special error value to be returned by h2o_quic_accept_cb, to indicate that packet decryption failed during quicly_accept
  */
-#define H2O_QUIC_ACCEPT_CONN_DECRYPTION_FAILED ((h2o_quic_conn_t *)1)
+#define H2O_QUIC_ACCEPT_CONN_DECRYPTION_FAILED (&h2o_quic_accept_conn_decryption_failed)
+/**
+ * special error value to be returned by h2o_http3_server_accept, indicating that a connection was accepted but already closed due
+ * to an error. In this case connection counter decrements are already done.
+ */
+#define H2O_QUIC_ACCEPT_CONN_CLOSED (&h2o_quic_accept_conn_closed)
```

Comment:
Can we get rid of these `#define`s and use the variables declared above?

At this moment, we do not need to take care of source compatibility, I do not think these macros are going to help otherwise.

Reply:
That's a right thing to do. Addressed in [9fc373f](https://github.com/h2o/h2o/pull/3024/commits/9fc373f3c5682d28a45516c65a7dac9189b44efc).